index.html

```diff
@@ -0,0 +1,167 @@
+                <div class="image">
+                    <span class="fa fa-music me"></span>
+                </div>
+                <p class="title1">Entertainment</p>
+            </div>
+            <p class="card-description ">  
+            <a href="#category-display" id="entertainment" class="clicable-card entertainment">These events are form of 
+                activities that holds the attention and interest of an 
+                audience or gives pleasure and delight.</a>
+            </p>
+         </div>
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa fa-book me"></span>
+                </div>
+                <p class="title1"> Education</p>
+            </div>
+            <p class="card-description "> 
+            <a href="#category-display" id="education" class="clicable-card education"> Events purposeful activity directed at achieving 
+                certain aims, such as transmitting 
+                knowledge or fostering skills and character traits</a>
+            </p>
+         </div>
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa-sharp fa-solid fa-basketball"></span>
+                </div>
+                <p class="title1"> Sports</p>
+            </div>
+            <p class="card-description ">  
+            <a href="#category-display" id="sports" class="clicable-card sports"> Events involving physical exertion and skill 
+                 in which an individual or team competes 
+                 against another or others for entertainment</a></p>
+         </div>
+        </div>
+        <a href="#category-display" class="see-category"> See Selected events category</a> 
+        </section> 
+      <!-- category-display card section---------------- -->
+      <section id="category-display"></section>  
+      <!-- other categories section-------------- -->
+      <section class="other-categories">
+        <h4> Other Categorie </h4>
+        <hr>
+        <ul>
+            <li> <span class="fa-regular fa-clock"></span> History</li> 
+            <li class="disable"><span class="fa-solid fa-dumbbell"></span> Fitness</li>
+            <li><span class="fa-solid fa-person-chalkboard"></span> Presentation</li>
+            <li class="disable"><span class="fa-solid fa-face-grin-tongue-wink"></span> Jokes</li>
+            <li><span class="fa-solid fa-person-falling-burst"></span> Comic</li>
+        </ul> 
+      </section>
+    </main>
+    <footer class="footer">
+        <div>
+        <a href="#index.html"> <img class="img" src="./images/logo.png" alt="logo"> 
+            <p class="logo"> Global <span> Streaming</span> </p></a>
+        </div>
+        <div class="footer-copyright container">
+            <p>&copy; 2022 Global Streaming</p>
+            <p>Author : Progress C. Ezeamaka</p>
+          </div>
+    </footer>
```

Comment:
- [ ] Please know that the mobile view of the home page shouldn't have "Other Categories" and a footer this is how yours look:
  ![image](https://user-images.githubusercontent.com/20567503/198013166-accf5bd9-c54c-4d5e-a37e-872795665b9d.png)
  And this is how the original design looks, it stops at the speakers' section:
  ![image](https://user-images.githubusercontent.com/20567503/198013289-d3590bc3-32f5-49a0-899a-ea2e7d1e052e.png)
  Kindly know that this is only for the mobile view, please fix it. 😁